autode/ext/src/points.cpp

```diff
@@ -172,10 +172,7 @@ namespace autode {
             // Should loop for all i, j and j, i but not i = j
             for (int j = 0; j < n; j++) {
 
-                // Only loop over non identical pairs
-                if (i == j) {
-                    continue;
-                }
+                if (i == j) continue;
```

Comment:
I was so confused for a second, but now I realise its not python...

Reply:
😂